publish.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash -e
 
-docker pull registry.tld/conjurinc/publish-rubygem
+set -e
+echo "==> Starting publish.sh"
 
+#echo "==> Removing Gemfile.lock"
+#rm -f Gemfile.lock
+
+echo "==> Cloning Release-tools"
+git clone git@github.com:conjurinc/release-tools.git
```

Comment:
Not sure if this should be the Atlantis repo?

Reply:
Cannot be done